examples/eks-cluster-with-observability/README.md

```diff
@@ -0,0 +1,215 @@
+# EKS Cluster with Amazon Managed Grafana and Amazon OpenSearch Service
```

Comment:
Observability pattern with EKS Cluster, Amazon Managed Prometheus, Amazon Managed Grafana and Amazon Open Search Service